docs/api.md

```diff
@@ -516,6 +521,12 @@ Obtain a `sessionToken` and, optionally, a `keyFetchToken` if `keys=true`.
   
   <!--end-request-body-post-accountlogin-metricsContext-->
 
+* `originalLoginEmail`: *validators.email.optional*
+
+  <!--begin-request-body-post-accountlogin-originalLoginEmail-->
+  
```

Comment:
Can you add a description in between the `begin` and `end` tags for this param and the ones below?